src/components/Table/Table.jsx

```diff
@@ -132,6 +132,10 @@ class Table extends React.Component {
     if (paginated) {
       data = data.slice(currentPage * pageLength, (currentPage + 1) * pageLength);
     }
+    data.forEach((row) => {
+      const r = row;
+      r.sortApplied = !!sortField && (sortField !== 'player_slot' || sortState !== 'asc');
```

Comment:
why the double negate?

Reply:
I use it instead of a type cast to boolean

Reply:
I prefer the type cast, it's more readable

Reply:
No idea what I was trying to do there, but it works just fine like this now. :)